src/helpers/getNightlyAdapter.ts

```diff
@@ -0,0 +1,18 @@
+import { NightlyConnectAdapter } from '@nightlylabs/wallet-selector-polkadot'
+
+let _adapter: NightlyConnectAdapter | undefined
+export const getNightlyConnectAdapter = async (persisted = true) => {
+  if (_adapter) return _adapter
+  _adapter = await NightlyConnectAdapter.build(
+    {
+      appMetadata: {
+        name: 'Use-inkathon selector',
+        description: 'Use-inkathon selector',
+        icon: 'https://inkathon.xyz/favicons/apple-touch-icon.png',
```

Comment:
This needs to be dynamic. Can you add `appName: string, appIcon?: string` to the function parameters in `getNightlyConnectAdapter`.